draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -8113,33 +8113,43 @@ Content-Range: exampleunit 11.2-14.3/25
    method. For the methods defined by this specification, the intended meaning
    of the payload can be summarized as:
 </t>
-<dl>
-  <dt>GET</dt>
-  <dd>
-    a representation of the <x:ref>target resource</x:ref>;
-  </dd>
-  <dt>HEAD</dt>
-  <dd>
-    the same representation as GET, but without the representation data;
-  </dd>
-  <dt>POST</dt>
-  <dd>
-    a representation of the status of, or results obtained from, the action;
-  </dd>
-  <dt>PUT, DELETE</dt>
-  <dd>
-    a representation of the status of the action;
-  </dd>
-  <dt>OPTIONS</dt>
-  <dd>
-    a representation of the communications options;
-  </dd>
-  <dt>TRACE</dt>
-  <dd>
-    a representation of the request message as received by the
-    end server.
-  </dd>
-</dl>
+<table align="left">
+  <thead>
+    <tr>
+      <th>method</th>
+      <th>meaning of the payload</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>GET</td>
+      <td>a representation of the <x:ref>target resource</x:ref></td>
+    </tr>
+    <tr>
+      <td>HEAD</td>
+      <td>the same representation as GET, but without transferring the
+          representation data</td>
+    </tr>
+    <tr>
+      <td>POST</td>
+      <td>a representation of the status of, or results obtained from,
+          the action</td>
+    </tr>
+    <tr>
+      <td>PUT, DELETE</td>
+      <td>a representation of the status of the action</td>
+    </tr>
+    <tr>
+      <td>OPTIONS</td>
+      <td>a representation of the communications options</td>
+    </tr>
+    <tr>
+      <td>TRACE</td>
+      <td>a representation of the request message as received by the
+          end server</td>
```

Comment:
"end server" seems a bit odd here, and I don't think it's a term we use anywhere else. Perhaps "server performing the trace"?

Reply:
```suggestion
          server returning the trace</td>
```